apps/server/src/routes/route_api.ts

```diff
@@ -158,6 +158,11 @@ function handleException(e: unknown | Error, method: HttpMethod, path: string, r
 
     log.error(`${method} ${path} threw exception: '${errMessage}', stack: ${errStack}`);
 
+    // Skip sending response if it's already been handled by the route handler
+    if ((res as unknown as { triliumResponseHandled?: boolean }).triliumResponseHandled || res.headersSent) {
```

Comment:
I need to be able to send my own responses when using the `asyncApiRoute` - so I figured this was the best way to go about it...